src/material-experimental/mdc-slide-toggle/slide-toggle.scss

```diff
@@ -92,6 +71,6 @@
     // Usually 1px would be enough, but MDC reduces the opacity on the
```

Comment:
The main `.mdc-switch` element no longer changes opacity. These lines should probably be removed, since they now lower the opacity of the switch:

```
  .mat-mdc-slide-toggle .mdc-switch--disabled {
    opacity: 0.75;
  }
```

That might also impact the decisions of this section, based on the comment